Detectors/FIT/FT0/workflow/src/ReconstructionSpec.cxx

```diff
@@ -65,7 +64,6 @@ void ReconstructionDPL::run(ProcessingContext& pc)
   pc.outputs().snapshot(Output{mOrigin, "RECCHDATA", 0, Lifetime::Timeframe}, mRecChData);
 
   mFinished = true;
-  pc.services().get<ControlService>().readyToQuit(QuitRequest::Me);
```

Comment:
why do you take this away?

Reply:
Actually, both the ``pc.services().get<ControlService>().readyToQuit(QuitRequest::Me)`` and ``mFinished = true;`` (which is never used elsewhere) should go away: the reconstruction should not decide that it finished its job, this is done by the upstream data supplier, which will ``ctx.services().get<o2f::ControlService>().endOfStream()`` once the data is over.

Reply:
Ok. Thanks for the explanation.